features/shared/environment.steps.ts

```diff
@@ -1,11 +1,27 @@
-import {Given, After} from '@cucumber/cucumber';
+import {execSync} from 'child_process';
+
+import {Given, After, Then} from 'cucumber';
 import tmp from 'tmp';
 import rimraf from 'rimraf';
+import path from 'pathe';
+import fs from 'fs-extra';
 
 Given('I have a working directory', function () {
   this.temporaryDirectory = tmp.dirSync().name;
 });
 
+Given(/I have the fixture (.+) in the working directory/, function (fixture) {
```

Comment:
Why are we using "fixture" and "feature"? Should we unify if we refer to the same?

Reply:
https://cucumber.io/docs/gherkin/reference/#feature

Reply:
A fixture represents a project that can be shared across multiple scenarios.